cypress/utils/filters.js

```diff
@@ -75,21 +78,37 @@ function applyFilters(filters, filtersConf) {
   }
 }
 
-function urlParamConvert(key, value) {
-  const filterCategory = _.find(
-    _.values(FILTER_CATEGORIES),
-    (it) => it.urlParam === key
-  );
-  const title = _.capitalize(filterCategory.title);
-  const label = _.find(filterCategory.values, (it) => it.value === value).label
-    .props.children;
-  return [title, label];
+function urlParamConvert(key, value, type) {
+  if (type === 'recommendation') {
+    const filterCategory = _.find(
+      _.values(FILTER_CATEGORIES),
```

Comment:
\+ to the Iker's comment

I think, instead of adding an extra prop to this function, you could better focus on `FILTER_CATEGORIES` and `CLUSTER_FILTER_CATEGORIES` (this is the only thing these two branches are different with, right?). Can we get back to the initial version of the function but only replace one line in the following way:

```suggestion
      _.values(FILTER_CATEGORIES.concat(CLUSTER_FILTER_CATEGORIES)),
```

Reply:
Maybe we can create an even more generic function, so that we pass the `FILTER_CATEGORIES` (in whichever form is required, i.e. `FILTER_CATEGORIES` or `CLUSTER_FILTER_CATEGORIES`). So the function will be something like:

```
function urlParamConvert(key, value, map) {
  const filterCategory = _.find(
    _.values(map),
    (it) => it.urlParam === key
  );
  const title = _.capitalize(filterCategory.title);
  const label = _.find(filterCategory.values, (it) => it.value === value).label
    .props.children;
  return [title, label];
}
```

and then we can create "partials" (sorry for the Python naming) for the different tests

```
urlParamConverter = (key, value) => urlParamConvert(key, value, FILTER_CATEGORIES)
```

Reply:
@ikerreyes, this is absolutely legit, from my point of view. Mine option is "the least effort," whereas yours is somewhat "generic/proper solution striving."